pkg/buildah/buildah.go

```diff
@@ -107,7 +108,7 @@ func RegisterGlobalFlags(fs *pflag.FlagSet) error {
 var (
 	globalFlagResults globalFlags
 	rootCmd           *cobra.Command
-	rootCmdName       string
+	unrelatedCommands = sets.NewString("version")
```

Comment:
`k8s.io/apimachinery/pkg/util/sets`'s `String` is deprecated. Use their new: `https://github.com/kubernetes/apimachinery/blob/master/pkg/util/sets/set.go`

Reply:
Maybe just new empty string is enough.(Suppress const vars.)

Reply:
OK